src/interfaces/libpq/ivytest/oracle_test.sh

```
@@ -0,0 +1,168 @@
+
+: ${MAKE=make}
+
+# Guard against parallel make issues (see comments in pg_regress.c)
+unset MAKEFLAGS
+unset MAKELEVEL
+
+# Run a given "initdb" binary and overlay the regression testing
+# authentication configuration.
+standard_initdb() {
+	initdb -D $PGDATA -USYSTEM -moracle
+	if [ -n "$TEMP_CONFIG" -a -r "$TEMP_CONFIG" ]
```

Shellcheck (0.10.0)

[warning] Prefer [ p ] && [ q ] as [ p -a q ] is not well defined.

(SC2166)